lib/dfe/reference_data/degrees/grades.rb

```diff
@@ -15,6 +16,7 @@ module Degrees
           '8741765a-13d8-4550-a413-c5a860a59d25' =>
           { hesa_code: '01',
             name: 'First-class honours',
+            abbreviation: '1',
```

Comment:
```suggestion
            abbreviation: 'First',
```

I don't think `1` makes sense as an abbreviation.